frontend/app/routes/_public+/intake+/$id+/review-information.tsx

```diff
@@ -0,0 +1,217 @@
+import type { ReactNode } from 'react';
+
+import { json, redirect } from '@remix-run/node';
+import type { ActionFunctionArgs, LoaderFunctionArgs } from '@remix-run/node';
+import { Form, useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+
+import { Address } from '~/components/address';
+import { Button } from '~/components/buttons';
+import { InlineLink } from '~/components/inline-link';
+import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('gcweb');
```

Comment:
You need to export this so that the client side loaders will see it and load the translations when doing a clientside nav..

ex:
``` typescript
export const handle: RouteHandleData = {
  i18nNamespaces,
  ...restOfHandle
} as const;
```

Reply:
```ts
...
} as const satisfies RouteHandleData;
```

Reply:
Why use satisfies over actually setting the type?

Reply:
> Why use satisfies over actually setting the type?

The purpose of satisfies is to enforce a constraint on a variable, without changing its type. ([source](https://dev.to/ayc0/typescript-49-satisfies-operator-1e4i))